src/OrchardCore.Modules/OrchardCore.Redis/HealthChecks/RedisHealthCheck.cs

```diff
@@ -22,7 +25,7 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             var redisService = _serviceProvider.GetService<IRedisService>();
             if (redisService == null)
             {
-                return HealthCheckResult.Unhealthy();
+                return HealthCheckResult.Unhealthy(description: S["The service '{0}' isn't registered.", nameof(IRedisService)]);
```

Comment:
I don't think these should be localized. We don't want to have to add localization support to the consumers of health checks; just because a site's language is not English, we should be able to check its health status in the same way.